### PR TITLE
Rough and ready mentoring docs

### PR DESCRIPTION
**Pull request by plaindocs**
@erin-rtfm was kind enough to tell me about her mentoring experience, and I've rephrased (lazily) so more new speakers can benefit. 

Improvements welcome.

CC @Bradamant3 @mjang @eirinikos

### CODE REVIEW

File: docs/organizer-guide/confs/cfp.rst [hunk 1]
```diff
@@ -140,3 +140,50 @@ Biases
...
+~~~~~~~~~~~~~~~~~
+
+Flexible according to the needs and availability of participants. Use the following guidelines if they are useful
+
+We met four times in the months leading up to the conference. Each meeting was an hour and there were about two to three weeks between each.
+
+Introduction/level-set:
+   Meet one another for the first time, I introduced my topic and experience with it, we decided on our meeting structure and a loose schedule. We agreed I would create an outline for my talk, then the slides, then present a practice talk to him.
+
+Present outline:j
```
Comment (ericholscher) [inline docs/organizer-guide/confs/cfp.rst]:
```suggestion
Present outline:
```

File: docs/organizer-guide/confs/cfp.rst [hunk 2]
```diff
@@ -140,3 +140,50 @@ Biases
...
+
+Meeting structure
+~~~~~~~~~~~~~~~~~
+
+Flexible according to the needs and availability of participants. Use the following guidelines if they are useful
+
+We met four times in the months leading up to the conference. Each meeting was an hour and there were about two to three weeks between each.
+
+Introduction/level-set:
+   Meet one another for the first time, I introduced my topic and experience with it, we decided on our meeting structure and a loose schedule. We agreed I would create an outline for my talk, then the slides, then present a practice talk to him.
```
Comment (ericholscher) [inline docs/organizer-guide/confs/cfp.rst]:
```suggestion
   Meet one another for the first time, I introduced my topic and experience with it, we decided on our meeting structure and a loose schedule. We agreed I would create an outline for my talk, then the slides, then present a practice talk to them.
```

Comment (ericholscher) [reply docs/organizer-guide/confs/cfp.rst]: There's a couple other places this is gendered. Not sure if we care or not.

**Review by ericholscher — APPROVED** [review]
Solid, concrete example that might help folks know what they're signing up for 👍

File: docs/organizer-guide/confs/cfp.rst (hunk 2 above)
Comment (plaindocs) [reply docs/organizer-guide/confs/cfp.rst]: Considering the new speaker / mentor dynamic we definitely care. Thanks for pointing out. <3

File: docs/organizer-guide/confs/cfp.rst [hunk 3]
```diff
@@ -140,3 +140,50 @@ Biases
...
+
+Flexible according to the needs and availability of participants. Use the following guidelines if they are useful
+
+We met four times in the months leading up to the conference. Each meeting was an hour and there were about two to three weeks between each.
+
+Introduction/level-set:
+   Meet one another for the first time, I introduced my topic and experience with it, we decided on our meeting structure and a loose schedule. We agreed I would create an outline for my talk, then the slides, then present a practice talk.
+
+Present outline:
+   In the two weeks between our introduction and this meeting, I prepped an outline for my talk, going into as much detail as I could. I presented it to my mentor and they asked me questions about the timings I expected and how I might structure my slides. They also presented his concerns around the order of my topics, how to build up the information I was sharing in layers.
```
Comment (ASRFS) [inline docs/organizer-guide/confs/cfp.rst]: They also presented his concerns --> They also presented their concerns

**Pull request merged**